portal/server/controllers/dashboard/facilities/facilities-filters-query.js

```diff
@@ -51,15 +55,28 @@ const dashboardFacilitiesFiltersQuery = (filters, user) => {
         keywordFilter.OR.push(...keywordFilters);
 
         query.AND.push(keywordFilter);
-      }
+      } else if (isStageField) {
+        const fieldFilter = {
+          OR: [],
+        };
+        filterValue.forEach((value) => {
+          if (value === FACILITY_STATUS.RISK_EXPIRED) {
+            fieldFilter.OR.push({ facilityStage: FACILITY_STATUS.RISK_EXPIRED });
+          } else {
+            fieldFilter.OR.push({
+              hasBeenIssued: value === BESPOKE_FILTER_VALUES.FACILITIES.ISSUED,
+            });
+          }
```

Comment:
this isnt the most extensible, when `Abandoned` is added in might want to refactor this

Reply:
I think leave for now though

Reply:
I was thinking the same thing about refactor -- Happy to think about it during abandon. Even if its just breaking down this large function into smaller ones, and revisiting the comments to make this easier for someone with no context dropping into this file!